lang/php/Portfile

```diff
@@ -4,6 +4,7 @@ PortSystem              1.0
 PortGroup               php 1.1
 PortGroup               compiler_blacklist_versions 1.0
 PortGroup               old_openssl 1.0
+PortGroup               select 1.0
```

Comment:
If we go with your proposal, then still not all of the subports need the select portgroup. Assuming that including the portgroup but not using it is not harmful, we would still probably want to restrict it to only a reasonable subset of the subports, such as the existing block handling all SAPI subports.

Reply:
I force pushed a change to only use the select group for php and php-cgi. The motivation for the change is that a certain PHP IDE will try to use php-cgi for the integrated dev server no matter what.